compiling on gcc 9

<b>System Information:</b>
  Arch linux
  gcc 9.1.0
_______________________________________________________________________________________

  <!-- Explain your issue/request/suggestion in detail here! -->
compiling fails with the following error:
```
In file included from /usr/include/qt/QtCore/QMap:1,
                 from /home/ebbit/repo/trice/Cockatrice/common/server_room.h:8,
                 from /home/ebbit/repo/trice/Cockatrice/common/server_room.cpp:1:
/usr/include/qt/QtCore/qmap.h: In instantiation of ‘QMap<K, V>::iterator QMap<K, V>::insert(const Key&, const T&) [with Key = QString; T = ServerInfo_User_Container]’:
/home/ebbit/repo/trice/Cockatrice/common/server_room.cpp:193:84:   required from here
/usr/include/qt/QtCore/qmap.h:723:25: error: implicitly-declared ‘ServerInfo_User_Container& ServerInfo_User_Container::operator=(const ServerInfo_User_Container&)’ is deprecated [-Werror=deprecated-copy]
  723 |         lastNode->value = avalue;
      |         ~~~~~~~~~~~~~~~~^~~~~~~~
In file included from /home/ebbit/repo/trice/Cockatrice/common/server_room.h:6,
                 from /home/ebbit/repo/trice/Cockatrice/common/server_room.cpp:1:
/home/ebbit/repo/trice/Cockatrice/common/serverinfo_user_container.h:14:5: note: because ‘ServerInfo_User_Container’ has user-provided ‘ServerInfo_User_Container::ServerInfo_User_Container(const ServerInfo_User_Container&)’
   14 |     ServerInfo_User_Container(const ServerInfo_User_Container &other);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~
```
The issue comes from somewhere in protobuf the operator= is missing or something.

Compiling with the flag `-DCMAKE_CXX_FLAGS="-Wno-error=deprecated-copy"` added to cmake solves it for now.
  <!-- This repository is ONLY about development of the Cockatrice app.
       If you have any problems with a server (e.g. registering, connecting, ban...)
       you have to contact that server's owner/admin. 
       Check this list of public servers with webpage links and contact details:
       https://github.com/Cockatrice/Cockatrice/wiki/Public-Servers -->

--- a/common/serverinfo_user_container.h
+++ b/common/serverinfo_user_container.h
@@ -12,6 +12,7 @@ public:
     ServerInfo_User_Container(ServerInfo_User *_userInfo = 0);
     ServerInfo_User_Container(const ServerInfo_User &_userInfo);
     ServerInfo_User_Container(const ServerInfo_User_Container &other);
+    ServerInfo_User_Container &operator=(const ServerInfo_User_Container &other) = default;
     virtual ~ServerInfo_User_Container();
     ServerInfo_User *getUserInfo() const
     {

